source/reporting/index.html.md.erb

```diff
@@ -0,0 +1,120 @@
+---
+title: Reporting 
+weight: 70
+---
+
+# Reporting 
+
+You can extract reporting information from GOV.UK Pay manually, or via the
+API.
+
+## Manual reporting
+
+You can sign in to your [GOV.UK Pay admin
+account](https://selfservice.payments.service.gov.uk/login) and export
+transaction information in a CSV file. 
+
+For a given account, select __Transactions__ in the __Dashboard__ page.
+
+You can filter transactions using the available search criteria, and then
+select __Download all transaction details as a CSV file__. 
+
+You can process CSV files using spreadsheet software.
+
+## Reporting via the GOV.UK Pay API 
+
+Before reading this section, you should read the section describing [Making
+payments](/making_payments). 
+
+You can develop software to get data from the GOV.UK Pay API and import it into 
+a finance system. This could remove routine manual operations from your workflow and
+lead to significant cost savings for your organisation. 
+
+You can use the GOV.UK Pay API to:
+
+* get information about a single payment
+* generate a list of payments matching search criteria  
+
+## Get information about a single payment
+
+`GET /v1/payments/paymentId`
+
+1. Sign in to <a href="https://gds-payments.gelato.io/api-explorer/"
+   target="blank">the API Explorer</a> (link opens in new window).  
+2. Under __Resource__, select __{Payment Id}. Under __Action__, select <a
+   href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2/general/"
+   target="blank">__Find payment by ID__</a> (link opens in new window). 
+   
+For a valid request, the JSON response body contains a `"state"` field. You
+can use this information when designing a service.  
+
+For example, the response for a successful payment contains:
+
+```
+  "state": {
+    "status": "success",
+    "finished": true
+    },
+```
+
+With a successful payment, you know that the user can continue their journey
+through your service.
+
+For a valid request, the JSON response body to this API call also contains
+other useful information. For example, if a user has gone far enough in their
+payment journey, the `"card_details"` section contains information that they
+have entered when making a payment: 
+
+```
+  "card_details": {
+    "last_digits_card_number": "1111",
+    "first_digits_card_number": "100002",
+    "cardholder_name": "Sherlock Holmes",
+    "expiry_date": "12/21",
+    "billing_address": {
+      "line1": "221 Baker Street",
+      "line2": "Flat b",
+      "postcode": "NW16XE",
+      "city": "London",
+      "country": "GB"
+    },
+```
+
+## Generate a list of payments (search payments) 
+
+`GET /v1/payments`
```

Comment:
This is fine (it will get all payments) — maybe we should explicitly call out that all payments are returned if no query parameters are provided

Reply:
> maybe we should explicitly call out that all payments are returned if no query parameters are provided

Sounds like a good idea.